src/features/user/ManageProjects/components/DetailedAnalysis.tsx

```diff
@@ -0,0 +1,306 @@
+import React, { ReactElement } from 'react'
+import MaterialTextField from '../../../common/InputTypes/MaterialTextField';
+import { useForm } from 'react-hook-form';
+import i18next from './../../../../../i18n'
+import ToggleSwitch from '../../../common/InputTypes/ToggleSwitch';
+import styles from './../styles/StepForm.module.scss'
+import AnimatedButton from '../../../common/InputTypes/AnimatedButton';
+import BackArrow from '../../../../../public/assets/images/icons/headerIcons/BackArrow';
+const { useTranslation } = i18next;
+interface Props {
+    handleNext: Function;
+    handleBack: Function;
+}
+
+export default function DetailedAnalysis({ handleBack, handleNext }: Props): ReactElement {
+    const { t, i18n } = useTranslation(['manageProjects']);
+
+    const { register, handleSubmit, errors } = useForm();
+
+    const [plantingSeasons, setPlantingSeasons] = React.useState([
+        { id: 0, title: 'January', isSet: false },
+        { id: 1, title: 'Febuary', isSet: false },
+        { id: 2, title: 'March', isSet: false },
+        { id: 3, title: 'April', isSet: false },
+        { id: 4, title: 'May', isSet: false },
+        { id: 5, title: 'June', isSet: false },
+        { id: 6, title: 'July', isSet: false },
+        { id: 7, title: 'August', isSet: false },
+        { id: 8, title: 'September', isSet: false },
+        { id: 9, title: 'October', isSet: false },
+        { id: 10, title: 'November', isSet: false },
+        { id: 11, title: 'December', isSet: false }
```

Comment:
Months names should either be retrieved from some date library (do we already use one to format dates in this project?) or added as translation resources.

Reply:
This will be done via translations